lib/error_notifier.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ErrorNotifier
+  class << self
+    def notify(exception, options = {})
+      debug_info = Samson::Hooks.fire(:exception, exception, options).compact.first
```

Comment:
might be better to do `.detect { |m| m.is_a?(String) }` to avoid caching things like the `false` from statsd.rb or whatever error handler does not support this feature